kola/tests/kubernetes/controllerInstall.go

```diff
@@ -0,0 +1,795 @@
+package kubernetes
+
+const controllerInstallScript = `#!/bin/bash
+set -e
+
+# List of etcd servers (http://ip:port), comma separated
+export ETCD_ENDPOINTS=
+
+# Specify the version (vX.Y.Z) of Kubernetes assets to deploy
+export K8S_VER=v1.1.3
```

Comment:
This ideally is parameterized in some way (also we very likely start by publishing images of v1.1.2, v1.1.4, and v1.1.7)